Bootstrap/Podfile

```diff
@@ -1,3 +1,5 @@
+source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
oh weird. Last night I was messing around with it, and it was giving me a warning on `pod install`.